README.md

```diff
@@ -43,6 +43,7 @@ nMigen can be used to target any FPGA or ASIC process that accepts behavioral Ve
 
   * Lattice iCE40 (toolchains: **Yosys+nextpnr**, LSE-iCECube2, Synplify-iCECube2);
   * Lattice MachXO2 (toolchains: Diamond);
+  * Lattice MachXO3l (toolchains: Diamond);
```

Comment:
Isn't the chip called `MachXO3L` (capital L)? E.g. in [vendor docs](http://www.latticesemi.com/Products/FPGAandCPLD/MachXO3.aspx).